.eslintrc

```diff
@@ -51,6 +51,7 @@
                 }
             }
         ],
+        "no-use-before-define": "off",
```

Comment:
`@typescript-eslint/no-use-before-define` is already covering this. After version bump this one results in some weird errors.